app/assets/javascripts/code/services/_draw_session.js

```diff
@@ -122,6 +122,4 @@ export const drawSession = (
   return new DrawSession();
 };
 
-function calculateHeatLevel(value) {
-  return heat.getLevel(value);
-};
+const calculateHeatLevel = (heat, value) => heat.getLevel(value);
```

Comment:
Just for the sake of philosophy this could be
```
const calculateHeatLevel = (heat, value) => heat.getLevel(value);
const calculateHeatLevel = heat => value => heat.getLevel(value);
const calculateHeatLevel = heat => heat.getLevel;
```
but I wouldn't go that far here